product_quick_bom/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module was written to create quickly the bom for your product. In the product form there is a new "bom" tab that gives the possibility to add a bom line directly.
```

Comment:
s/bom/BOM/
(acronyms are better with uppercase characters)

Reply:
in fact, it's BoM for Bill of Material in Odoo ;-)
(like PoS for Point of Sale)